backend/src/main/java/com/zeros/backend/config/SwaggerConfig.java

```diff
@@ -0,0 +1,54 @@
+package com.zeros.backend.config;
+
+import com.google.common.base.Predicate;
+import com.google.common.base.Predicates;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.bind.annotation.RequestMethod;
+import springfox.documentation.RequestHandler;
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.service.ApiInfo;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+import java.util.ArrayList;
+import java.util.Set;
+
+@Configuration
+@EnableSwagger2
+public class SwaggerConfig {
```

Comment:
I wonder if we need to add a profile to this class to disable it when deployed. What do you think @JumperSpirit ?